prowler/providers/aws/services/athena/athena_service.py

```diff
@@ -113,15 +109,10 @@ def _list_tags_for_resource(self):
         logger.info("Athena - Listing Tags...")
         try:
             for workgroup in self.workgroups.values():
-                try:
-                    regional_client = self.regional_clients[workgroup.region]
-                    workgroup.tags = regional_client.list_tags_for_resource(
-                        ResourceARN=workgroup.arn
-                    )["Tags"]
-                except Exception as error:
-                    logger.error(
-                        f"{self.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
-                    )
+                regional_client = self.regional_clients[workgroup.region]
```

Comment:
Why did you remove this try?
This is to continue the next iterations in case the previous one failed.